### PR TITLE
PRIME-2614 Update login page

### PR DESCRIPTION
**Pull request by bergomi02**

### CODE REVIEW

File: prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { ConfirmDialogComponent } from '@shared/components/dialogs/confirm-dialo
 import { HtmlComponent } from '@shared/components/dialogs/content/html/html.component';
 import { BannerLocationCode } from '@shared/enums/banner-location-code.enum';
 import { CollectionNoticeService } from '@shared/services/collection-notice.service';
+import { MAT_RADIO_DEFAULT_OPTIONS, MAT_RADIO_GROUP_CONTROL_VALUE_ACCESSOR, MatRadioChange } from '@angular/material/radio';
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.ts:14]: Please remove unused imports

File: prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html [hunk 2]
```diff
@@ -62,6 +62,10 @@
     <div class="mid-content">
       <div class="mt1">Request approval to access PharmaNet</div>
       <div class="mt1">Renew or update existing enrolment</div>
+      <div class="button_title">
+        Health care providers working in health authority faciliites, private community health practice sites, community
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html:66]: Typo in "faciliites"

File: prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.ts (hunk 1 above)
Comment (bergomi02) [reply prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.ts]: it is fixed

File: prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html (hunk 2 above)
Comment (bergomi02) [reply prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html]: it is fixed

**Review by neophyte57 — APPROVED** [review]
Nicely done  😊

**Pull request merged**